### ISSUE TITLE
Add color adaptation utility function (contrast util)

### ISSUE THREAD

**Issue by gtyrkicksin216** [fulcrumui/css-utils#2]
Color contrast function needs to be added to allow consumers to be able to easily utilize this on elements such as buttons and chips. This function will take the background color provided and ensure that either white/black are used as the text color to ensure proper accessibility contrast standards are met.
